src/components/GlobalFilter/GlobalFilter.tsx

```diff
@@ -54,18 +49,7 @@ const useLoadTags = (hasAccess = false) => {
 
 const GlobalFilter = ({ hasAccess }: { hasAccess: boolean }) => {
   const [isOpen, setIsOpen] = useState(false);
-  const dispatch = useDispatch();
-  const isLoaded = useSelector(({ globalFilter: { tags, sid, workloads } }: ReduxState) => tags.isLoaded && sid.isLoaded && workloads.isLoaded);
-  const { count, total, tags, sid, workloads } = useSelector(
-    ({ globalFilter: { tags, sid, workloads } }: ReduxState) => ({
-      count: (tags.count || 0) + (sid.count || 0) + (workloads.count || 0),
-      total: (tags.total || 0) + (sid.total || 0) + (workloads.total || 0),
-      tags: tags.items || [],
-      sid: sid.items || [],
-      workloads,
-    }),
-    shallowEqual
-  );
+  const { count, total, tags, sid, workloads, isLoaded } = useGlobalFilterState();
```

Comment:
Created a hook for this, may be unnecessary?

Reply:
When consolidating state variables, just be careful you are properly memorizing the state output. To prevent extra re-renders.

I don't see an issue with using the jotai atoms directly.